slf4j-api/src/main/java/org/slf4j/impl/StaticMDCBinder.java

```diff
@@ -37,7 +37,16 @@
     /**
      * The unique instance of this class.
      */
-    public static final StaticMDCBinder SINGLETON = new StaticMDCBinder();
```

Comment:
I'm not sure about making this private right now. Perhaps mark it as @Deprecated instead so people know to switch to accessor method.